src/agoradatatools/gx.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import shutil
+
+from synapseclient import File, Synapse
+
+import great_expectations as gx
+from great_expectations.checkpoint.types.checkpoint_result import CheckpointResult
+
+logger = logging.getLogger(__name__)
+# Disable GX INFO logging
+logging.getLogger("great_expectations").setLevel(logging.WARNING)
+
+
+class GreatExpectationsRunner:
+    """Class to run great expectations on a dataset and upload the HTML report to Synapse"""
+
+    gx_project_dir = "./great_expectations"
```

Comment:
nit: the risk to having a relative path here is based on where the execution of the code happens.  Maybe something like this: https://stackoverflow.com/questions/4934806/how-can-i-find-scripts-directory?

The caveat to that is then it makes it difficult to test the code in an IDE.

Reply:
I'm still a little confused on this point. Doing something like `os.path.dirname(os.path.realpath(__file__))` to get the directory the script is in works fine, but do I then need to go up from that directory by two levels and add the `/great_expectations` directory on the end to get my `gx_project_dir`? So it's still a relative path, but confirmed to be relative to the location of the script?

Reply:
I added something like this in my latest commit

Reply:
I think you can just do

os.path.join(scriptdir, "../great expectations")

Reply:
Ah, I did not realize you could do that. Thanks!

Reply:
Unfortunately, it seems like this still [does not work](https://github.com/Sage-Bionetworks/agora-data-tools/actions/runs/6893527959/job/18753299928#step:5:445) in GH Actions. This error means that it is failing to find the `great_expectations` directory.

Reply:
When I run this locally, the path is
`/Users/bmacdonald/Development/repos/agora-data-tools/src/agoradatatools/../../great_expectations`
But when it runs in the GH runner it is 
`/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/agoradatatools/../../great_expectations`

Reply:
Ah... thats because it's out of the `src` folder. hmmm, what are your thoughts about adding it in the `src` folder?

Reply:
I suppose it's necessary. I don't think there is much of a difference between the locations in terms of maintainability

Reply:
I tried putting it in [src](https://github.com/Sage-Bionetworks/agora-data-tools/actions/runs/6893936077), and [agoradatatools](https://github.com/Sage-Bionetworks/agora-data-tools/actions/runs/6893975989), both failed in GH Actions.

Reply:
It looks like the `great_expectations` folder is currently not ending up in the package folder when installed:
```
Run ls /opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/agoradatatools
__init__.py
__main__.py
__pycache__
errors.py
etl
gx.py
logs.py
process.py
```

Reply:
I was able to resolve this problem with the introduction of a `MANIFEST.in` file which instructs the `great_expectations` folder and its contents to be included when this package is installed.